scalafix-core/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -61,7 +61,16 @@ sealed abstract class Patch {
   def +(other: Option[Patch]): Patch =
     this.+(other.getOrElse(Patch.empty))
   def ++(other: Iterable[Patch]): Patch = other.foldLeft(this)(_ + _)
+
+  /**
+    * Check if the patch is Patch.empty. This is an inexpensive computation.
```

Comment:
The main point of this operation is not that it's cheap so I would rather formulate it this way

> Returns true if this patch is `Patch.empty`, false otherwise.
>
> Note, may return false for patches that produce empty diffs. For example, a `Patch.replaceSymbols(from, to)` returns false even if the symbol `from` is not referenced in the code resulting in an empty diff.